sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/meta/provider/bigquery/BigQueryTableProviderTest.java

```diff
@@ -67,7 +67,7 @@ public void testDefaultMethod_whenPropertiesAreNotSet() {
   public void testSelectDefaultMethodExplicitly() {
     Table table =
         fakeTableWithProperties(
-            "hello", "{ " + METHOD_PROPERTY + ": " + "\"" + Method.DEFAULT.toString() + "\" }");
+            "hello", "{\"" + METHOD_PROPERTY + "\": " + "\"" + Method.DEFAULT.toString() + "\" }");
```

Comment:
there are multiple changes of adding quotes here. Do they mean a breaking change (SQL statements worked before now need to add quotes) / is there any chance keep the original semantics?

Reply:
I will fix it, with JsonParser.Feature.ALLOW_UNQUOTED_FIELD_NAMES true